### PR TITLE
Support PHP 8.2

### PR DESCRIPTION
**Pull request by maxnz**
Second attempt because #28 wasn't working

### CODE REVIEW

**Comment by AaronAtDuo** [comment]
Ok, something must be wrong with our CI.  I'll try to figure it out today.

**Comment by AaronAtDuo** [comment]
Looks like Actions may be down, it's red on the status page at this moment
https://www.githubstatus.com/

**Comment by maxnz** [comment]
Looks like it just needs approval to start now

**Comment by AaronAtDuo** [comment]
I can't seem to get it to run for 8.1 and 8.2, it cancels those jobs due to the other failures; so I'm not sure if those would pass... but it looks like there's some dependency version issues to resolve?

**Comment by maxnz** [comment]
I think that's what I found when I ran the CI on my forked copy a while ago. I don't know enough about the direction of the project to know how to resolve those

**Comment by AaronAtDuo** [comment]
In general, we try to keep up to date on the currently supported versions of the language.  So per https://www.php.net/supported-versions.php we can probably stop testing on 7.4.  But we'd still want 8.0 to work.  I'll see if I can take a look at the specific problem dependencies and figure out what we want.

**Comment by AaronAtDuo** [comment]
I was able to get your PR working against both 8.0 and 8.2 with the attached composer.lock file - if you use this and rebase, let's see how CI goes :crossed_fingers: ?
[composer.lock.txt](https://github.com/duosecurity/duo_api_php/files/11479333/composer.lock.txt)

**Comment by maxnz** [comment]
Looks like the tests passed on my fork

**Comment by maxnz** [comment]
Any updates?

File: README.md [hunk 1]
```diff
@@ -12,6 +12,7 @@
 * 7.4
```
Comment (yizshi) [inline README.md:12]: small nitpicking,
we should remove 7.4 here too

**Comment by AaronAtDuo** [comment]
@maxnz I apologize for the delays; I've not been able to stay as on-top of Github stuff as I would like.  Anyway, I'll merge this in now.  I really appreciate your work and patience on this.

**Comment by maxnz** [comment]
No worries, glad to help!

**Comment by maxnz** [comment]
Will a 1.0.4 or 1.1.0 tag be added?

**Comment by AaronAtDuo** [comment]
Yes, I'm planning to do a release Monday
(Painful experience has taught me not to release on a Friday :) )

**Comment by AaronAtDuo** [comment]
@maxnz https://packagist.org/packages/duosecurity/duo_api_php#1.1.0 hot off the presses.  thanks again for your help.

**Pull request merged**